docs/src/main/sphinx/release/release-463.md

```diff
@@ -0,0 +1,27 @@
+# Release 463 (23 Oct 2024)
+
+## General
+
+* Add {func}`timezone` functions to extract the timezone identifier from a
+  timestamp or time. ({issue}`20893`)
```

Comment:
Maybe say "from a `timestamp(p) with time zone` or `time(p) with time zone`.

Reply:
I was contemplating .. lets add that detail.